shell.nix

```diff
@@ -33,6 +33,7 @@ let
 
     # integration testing utilities
     python310Packages.pip
```

Comment:
You can add black to the requirements.txt file..  so that the IDE can be configured to apply format automatically.

Reply:
ok I updated it